test/utils.test.js

```diff
@@ -0,0 +1,7 @@
+const utils = require('../src/utils');
+
+test('test for URL validation', () => {
```

Comment:
Let's break these up into separate tests with a description for each of what is tested.